src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import styles from "./Footer.module.scss";
+import React from "react";
+import {FiFacebook} from "react-icons/fi"
```

Comment:
You are importing two things from the same library "react-icons/fi", merge the imports, i.e. import them in a single line.